apps/emqx_auth_jwt/src/emqx_auth_jwt_svr.erl

```diff
@@ -204,13 +204,13 @@ do_verify(JwsCompacted, [Jwk|More]) ->
 check_claims(Claims) ->
     Now = os:system_time(seconds),
     Checker = [{<<"exp">>, fun(ExpireTime) ->
-                               Now < ExpireTime
+                               is_integer(ExpireTime) andalso Now < ExpireTime
```

Comment:
This changes existing behaviour, but the old version seems to have a vulnerability.
Previously, JWTs with noninteger `exp` values, like `"exp": "0"`, were always considered valid.

Reply:
to be inclusive for timestamps like "1651847033", make sense to convert from string to integer and check ?

Reply:
✅